### ISSUE TITLE
Cannot read property 'uids' of undefined

### ISSUE THREAD

**Issue by slarti-b** [microsoft/AzureStorageExplorer#1566]
**Storage Explorer Version**: 1.9.0
**Build Number**: 20190701.8
**Platform/OS**:  Windows 10
**Architecture**: x64

## Bug Description ##
Installed Sotrage Explorer and opened for the first time.  it found my corporate account but said _"Subscriptions could not be loaded"_ (which is fine - I don't have an azure accounted conencted to that account). Went to _Add Account_ and chose _Use a storage account name and key_ then entered said info.  It showed an error _"Cannot read property 'uids' of undefined"_.  Turns out it had conencted the account, but didn't see that initially.

## Steps to Reproduce ##
1. Fresh install of Storage Explorer
2. Click Add Account
3. Chose Use a storage account name and key
4. Enter name and key

## Expected Experience ##
If the account connection I was trying to make succeeds then i expect it to say that and maybe show me that connection.

## Actual Experience ##
I got the following error.  I tried adding it again at which point it said it already existed and I browsed around and found it. 
```
{
  "name": "TypeError",
  "message": "Cannot read property 'uids' of undefined",
  "__zone_symbol__error": {
    "originalStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:172:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:76:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:99:76)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)",
    "zoneAwareStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:172:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:76:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:99:76)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)"
  },
  "stack": "TypeError: Cannot read property 'uids' of undefined\n    at StorageExplorerProvider.<anonymous> (file:///C:/Program%20Files%20(x86)/Microsoft%20Azure%20Storage%20Explorer/resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:282:51)\n    at step (file:///C:/Program%20Files%20(x86)/Microsoft%20Azure%20Storage%20Explorer/resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:47:23)\n    at Object.next (file:///C:/Program%20Files%20(x86)/Microsoft%20Azure%20Storage%20Explorer/resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:28:53)\n    at fulfilled (file:///C:/Program%20Files%20(x86)/Microsoft%20Azure%20Storage%20Explorer/resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:19:58)\n    at <anonymous>",
  "originalStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:172:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:76:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:99:76)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)",
  "zoneAwareStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:172:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:76:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:99:76)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)",
  "__zone_symbol__stack": "TypeError: Cannot read property 'uids' of undefined\n    at StorageExplorerProvider.<anonymous> (file:///C:/Program%20Files%20(x86)/Microsoft%20Azure%20Storage%20Explorer/resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:282:51)\n    at step (file:///C:/Program%20Files%20(x86)/Microsoft%20Azure%20Storage%20Explorer/resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:47:23)\n    at Object.next (file:///C:/Program%20Files%20(x86)/Microsoft%20Azure%20Storage%20Explorer/resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:28:53)\n    at fulfilled (file:///C:/Program%20Files%20(x86)/Microsoft%20Azure%20Storage%20Explorer/resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:19:58)\n    at <anonymous>",
  "__zone_symbol__name": "TypeError",
  "__zone_symbol__message": "Cannot read property 'uids' of undefined",
  "__zone_symbol__originalStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:172:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:76:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:99:76)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)",
  "__zone_symbol__zoneAwareStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:172:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:76:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:99:76)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)"
}
```

**Comment by Straightly** [comment]
Same issue.  Strangely, on my home computer, this worked out just fine.

**Comment by JasonYeMSFT** [comment]
Hi all,

When you are seeing this error, does you left hand side panel gets focused at the tree of resources or at the account panel? We are aware that Storage Explorer is throwing this error incorrectly when your left hand side gets focused at the account panel. It is actually safe to ignore it for now and keep using.

Thanks!

**Comment by kulmandy22** [comment]
Looks like this is some bug with Explorer. I also faced the same error. However if you go to Explorer tree view and try to hit "Refresh All" you can see the storage connected under Storage Accounts. 

Hope this helps. 

Thanks

**Comment by Straightly** [comment]
The account was not connected for me.  I wouldn’t mind the error at all
otherwise.  If it matters, I was getting this error inside a firewall and
can connect with everything the same from an Azure VM, which is of course
out of our Corp firewall.

**Comment by JasonYeMSFT** [comment]
Hi @Straightly ,
What exact error are you seeing when you expand the storage account node that you were trying to attach? For our known "uid" issue, it should not block you from connecting to the storage account. If this is still blocking you, can you reproduce this issue and turn on the developer tools and share any error messages you see with us?

1. Attach the storage account
2. Expand the attached account node
![image](https://user-images.githubusercontent.com/39359541/61834751-fba27480-ae2d-11e9-8598-509f0a939f32.png)
3. Open developer tools in the help menu and copy the error messages in the console panel at the bottom of the window.

Thanks!

**Comment by bpipal** [comment]
@JasonYeMSFT  I have this same problem on local here is what I see. 

![Capture](https://user-images.githubusercontent.com/656734/61835165-83d54980-ae2f-11e9-8cfa-d548ad8094d8.PNG)

I get this error when trying to expand it.

![image](https://user-images.githubusercontent.com/656734/61835201-a8312600-ae2f-11e9-8ffb-658651ba295f.png)

Note I'm MSFT FTE if you want to ping me off-line

**Comment by MRayermannMSFT** [comment]
@bpipal , that looks like an error from your emulator. We suggest reinstalling and restarting your emulator for that. And then if you continue to have emulator issues, please open a new issue. :)

**Comment by bpipal** [comment]
thanks, note I had the same exact problem as this issue is talking about, it said it failed to create the connection but it did, when expanding it, it gave me this new error. I'll create a new issue if this continues.

**Comment by JasonYeMSFT** [comment]
For "Cannot read property 'uid' of undefined" issue, it has been fixed in #1532 . 
Please open a new issue if yours is not the same as that one.
